stream-chat-android-ui-components/src/main/res/values/colors.xml

```diff
@@ -62,4 +62,23 @@
     <color name="stream_ui_input_message_send_button">#0076FF</color>
 
     <color name="stream_ui_channel_item_text_color">@color/stream_ui_black</color>
+    <color name="stream_ui_disabled_send_message_dark_theme">#2D2E2E</color>
+
+    <color name="stream_ui_background_default">@color/stream_ui_white</color>
+    <color name="stream_ui_background_light">@color/stream_ui_white</color>
+    <color name="stream_ui_background_alabaster">@color/stream_ui_alabaster</color>
+    <color name="stream_ui_background_alabaster_black">@color/stream_ui_alabaster</color>
```

Comment:
I get that this is "alabaster, or black when in night mode", but its name is a bit odd this way